modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -228,141 +372,38 @@ The behavior of `IF NOT EXISTS` and `OR REPLACE` apply to both standard and comp
-----
-
-
-==== Start a database with xref:database-administration/standard-databases/wait-options.adoc[`WAIT`]
-
-You can start your database using `WAIT` sub-clause to ensure that the command waits for a specified amount of time until the database is started.
-
-[source, cypher]
-----
-START DATABASE customers WAIT 5 SECONDS
-----
-
-
-[[manage-databases-stop]]
-== Stop databases
-
-Databases can be stopped using the command `STOP DATABASE`.
-
-=== Syntax
-
-[options="header", width="100%", cols="1m,5a"]
+[role="queryresult",options="header,footer",cols="2*<m"]
 |===
-| Command | Syntax
+| name | defaultLanguage
 
-| STOP DATABASE
-|
-[source, syntax, role="noheader"]
-----
-STOP DATABASE name [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
-----
+| "actors"  | "CYPHER 25"
+| "movies"  | "CYPHER 5"
+| "neo4j"   | "CYPHER 25"
+| "system"  | "CYPHER 25"
 
+2+d|Rows: 4
 |===
 
-=== Examples
-
-==== Stop a database
-
-To stop a database, use the following command:
-
-[source, cypher]
-----
-STOP DATABASE customers
-----
+For more information about other options for configuring the Cypher version, see xref:configuration/cypher-version-configuration.adoc[Configure the Cypher default version].
 
 [NOTE]
 ====
-Both standard databases and composite databases can be stopped using this command.
-====
-
-The status of the stopped database can be seen using the command `SHOW DATABASE name`:
-
-[source, cypher]
-----
-SHOW DATABASE customers YIELD name, requestedStatus, currentStatus
-----
-
-.Result
-[role="queryresult"]
-----
-+-----------------------------------------------+
-| name        | requestedStatus | currentStatus |
-+-----------------------------------------------+
-| "customers" | "offline"       | "offline"     |
-+-----------------------------------------------+
-----
-
-==== Stop a database with xref:database-administration/standard-databases/wait-options.adoc[`WAIT`]
-
-You can also stop your database using the `WAIT` sub-clause, which allows you to specify the amount of time that the system should wait for the database to stop.
-
-[source, cypher]
-----
-STOP DATABASE customers WAIT 10 SECONDS
-----
-
-
-[NOTE]
-====
-Databases that are stopped with the `STOP` command are completely shut down and may be started again through the `START` command.
-In a cluster, as long as a database is in a shutdown state, it can not be considered available to other members of the cluster.
-It is not possible to do online backups against shutdown databases and they need to be taken into special consideration during disaster recovery, as they do not have a running Raft machine while shutdown.
-Unlike stopped databases, dropped databases are completely removed and are not intended to be used again at all.
-====
+Setting the default language to `CYPHER 25` ensures that all queries run on that database will use the version of `Cypher 25` that the database is currently running (unless you prepend your queries with `CYPHER 5`, which overrides this default).
```

Comment:
TBH, I don't understand the phrase 'all queries run on that database will use **the version of `Cypher 25` that the database is currently running**'. 

Probably the idea is to show that Cypher25 will differ depending on neo4j versions.

Reply:
Yes, that's what it means.